ddtrace/contrib/falcon/middleware.py

```diff
@@ -12,20 +11,15 @@
 
 
 class TraceMiddleware(object):
-    def __init__(self, tracer, service="falcon", distributed_tracing=True):
```

Comment:
We still need this though, we can just set the integration config directly if it is set 🙂

Reply:
I see, thanks for the catch! Fixed appropriately 😄